odoo/tools/template_inheritance.py

```diff
@@ -219,12 +219,9 @@ def extract(spec):
                     node.addprevious(child)
             elif pos == 'content':
                 ''' Replace the entire content of an element '''
-                target = copy.deepcopy(node)
-                # Clear the content and attributes
-                node.clear()
-                # Get back attributes
-                for attr in target.keys():
-                    node.set(attr, target.get(attr))
+                for child in node:
```

Comment:
Start with `children = list(node)` as the doc suggests and use `children` for the rest of your implemantation, probably better and the code will be easier to read.